src/test/resources/network/brightspots/rcv/test_data/stop_tabulation_early_test/stop_tabulation_early_test_expected_summary.json

```diff
@@ -0,0 +1,33 @@
+{
+  "config" : {
+    "contest" : "Stop Tabulation Early Test",
+    "date" : "2023-03-14",
+    "jurisdiction" : "Funkytown, USA",
+    "office" : "Sergeant-at-Arms",
+    "threshold" : "4"
+  },
+  "results" : [ {
+    "round" : 1,
+    "tally" : {
+      "George Gervin" : "3",
+      "Mookie Blaylock" : "3",
+      "Yinka Dare" : "3"
+    },
+    "tallyResults" : [ {
+      "eliminated" : "Yinka Dare",
+      "transfers" : {
+        "exhausted" : "3"
+      }
+    } ]
+  }, {
+    "round" : 2,
+    "tally" : {
+      "George Gervin" : "3",
+      "Mookie Blaylock" : "3"
+    },
+    "tallyResults" : [ {
+      "eliminated" : "George Gervin",
+      "transfers" : { }
```

Comment:
Reading the legislation, having no transfers on this round seems correct. Let me know if anybody disagrees. (It doesn't make sense to add in last-round transfers to me, but I did want to flag just in case.)

Reply:
I'm inclined to agree, since transfers would only be relevant if you were going to then proceed and show the tallies for the next round. But @chughes297 and @rrojas350 may have an opinion.